### PR TITLE
vapor_master: 0.1.0-0 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by NickZ**
Increasing version of package(s) in repository `vapor_master` to `0.1.0-0`:

- upstream repository: https://github.com/roshub/vapor_master.git
- release repository: https://github.com/roshub/vapor_master-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.6.9`
- previous version of package: `null`

### vapor_master
```
* initial release
```

### CODE REVIEW

**Review by clalancette — APPROVED** [review]
This looks OK to me, though I'd like to get another opinion before merging.

One suggestion I have is to add license information to the top of every source file in the source repository (like https://github.com/roshub/vapor_master/blob/master/src/config.js).  I don't think it is strictly necessary (the LICENSE file at the top should suffice), but it is standard practice.

**Comment by tfoote** [comment]
Including the header would be good though I know that js is often optimized to avoid content like that. But since the project license is clearly and consistently indicated at the top level it's good for us to merge.

**Comment by tfoote** [comment]
The stretch builds are fialing due to npm not being in stretch: See #19845

**Pull request merged**
